flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveSchedulerTest.java

```diff
@@ -1401,7 +1403,9 @@ public void testTryToAssignSlotsReturnsNotPossibleIfExpectedResourcesAreNotAvail
                 adaptiveScheduler.tryToAssignSlots(
                         CreatingExecutionGraph.ExecutionGraphWithVertexParallelism.create(
                                 new StateTrackingMockExecutionGraph(),
-                                new CreatingExecutionGraphTest.TestingVertexParallelism()));
+                                new JobSchedulingPlan(
+                                        new VertexParallelism(Collections.emptyMap()),
```

Comment:
nit, would it make sense to introduce `VertexParallelism.empty()`? I've seen this construct in more places.

Reply:
Done in 1953ef9a6f295b77b410e96de9649cace1aaad34.